### PR TITLE
Remove support for user.parent-uid = 0

### PR DESCRIPTION
**Pull request by MrPetovan**
uid = 0 is the system user which isn't supposed to be the parent-uid of all the non-delegate users

See https://friendica.opensocial.space/display/8ffdf77c-1664-70c0-e908-261313207977

### CODE REVIEW

**Comment by annando** [comment]
Currently the tests are failing for some reason. Can you have a look at it? Also: Have you tested it on your system?

**Comment by MrPetovan** [comment]
I'm already testing the group => circle branch, I can't test both at the same time.

Edit: Just saw you merged the PR, I'll move this branch to my node next.

**Comment by MrPetovan** [comment]
This branch has now ran successfully on my dev node, there's no `parent-uid = 0` in the user table anymore, `User::getAdminListForEmailing` returns the expected data and delegation works.

File: database.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2023.09-dev (Giant Rhubarb)
--- DB_UPDATE_VERSION 1518
+-- DB_UPDATE_VERSION 1519
```
Comment (annando) [inline database.sql]: Remember to increase the database version number.

Comment (MrPetovan) [reply database.sql]: Of course you could have also merged this PR before yours!

**Comment by MrPetovan** [comment]
PR updated

**Pull request merged**